### ISSUE TITLE
How to get the none-reference sequence fasta for the following analysis

### ISSUE THREAD

**Issue by ld9866** [WarrenLab/chicken-pangenome-paper#1]
Dear authors:
Congratulations on publishing an excellent pan genomic work! We are currently using MC to construct the pan genome, but we are not very clear on how to use the pan genome to obtain non reference sequences. I would like to ask how we can step by step replicate your code in the article?

**Comment by esrice** [comment]
I'm not sure what you mean by using the pangenome to obtain non-reference
sequence. All code used in the paper should be in this repository; if
there's something missing let us know where exactly in the paper this
missing part is discussed and we will fix it right away.

**Comment by ld9866** [comment]
Dear authors:
We noticed that in the result part "Creation of pangenome references" there is a description "Therefore, there is in total 0.11 Gb (9.9%) of additional sequence in the graph compared to the total length of the nodes traversed by the most complete assembly. Of the other assemblies, bGalGal1b contributes the most additional sequence, 55.6 Mb, to the graph, whereas some assemblies contribute as little as 200 kb of additional sequence as a result of their relatedness to others"
First, we used our data with the sequence_per_sample python shell which works well, but we found that our reference genome is on the last line, we do not understand what happened.
Result:
sample1	2410672516
sample2	145612199
sample3	128427389
sample4	117406044
sample5	109404269
sample6	91436684
sample7	68120455
sample8	65357060
sample9	58917412
sample10	48846276
sample11	40840399
sample12	36126215
sample13	31310360
sample14	24277488
sample15	23999682
sample16	21216724
sample17	20872495
sample18	19711484
sample19	18913941
sample20	17094049
sample21	16712469
sample22	16466037
sample23	16023028
sample24	11992265
sample25	9518051
sample26	9410183
reference	1537762

Another question is whether we can obtain specific information about these sequences in other individuals, such as the 55.6Mb sequence of bGalGal1b, it would also be great if we obtain this information.
